libs/UIHelpers/transfer_function_widget.cpp

```diff
@@ -135,135 +135,156 @@ void TransferFunctionWidget::draw_ui()
   draw_list->AddRect(canvas_pos, canvas_pos + canvas_size, ImColor(180, 180, 180, 255));
 
   ImGui::InvisibleButton("tfn_canvas", canvas_size);
-  if (ImGui::IsItemHovered() || selected_point != (size_t)-1) {
+  if (ImGui::IsItemHovered() || selected_point != (size_t)-1)
+  {
 
     vec2f mouse_pos = (vec2f(io.MousePos) - view_offset) / view_scale;
     mouse_pos.x = clamp(mouse_pos.x, 0.f, 1.f);
     mouse_pos.y = clamp(mouse_pos.y, 0.f, 1.f);
 
-
-    if (io.MouseDown[0]) {
-      if (selected_point != (size_t)-1) {
+    if (io.MouseDown[0])
+    {
+      if (selected_point != (size_t)-1)
+      {
         alpha_control_pts[selected_point] = mouse_pos;
 
-        //make sure point does not cross
+        // make sure point does not cross
 
-          // Keep the first and last control points at the edges
-        if (selected_point == 0) {
+        // Keep the first and last control points at the edges
+        if (selected_point == 0)
+        {
           alpha_control_pts[selected_point].x = 0.f;
         }
-        else if (selected_point == alpha_control_pts.size() - 1) {
+        else if (selected_point == alpha_control_pts.size() - 1)
+        {
           alpha_control_pts[selected_point].x = 1.f;
         }
-        //make sure the other points do not cross
+        // make sure the other points do not cross
         else
         {
           alpha_control_pts[selected_point].x = (alpha_control_pts[selected_point].x < alpha_control_pts[selected_point - 1].x)
-            ? alpha_control_pts[selected_point - 1].x + 0.01 : alpha_control_pts[selected_point].x;
+                                                    ? alpha_control_pts[selected_point - 1].x + 0.01
+                                                    : alpha_control_pts[selected_point].x;
```

Comment:
Yeah I'm really curious what's going on with the formatter. The ternary operator is lined up but it seems to be at a random indent. I think it's worth it to run the formatter again to see what it's spitting out

Reply:
maybe the line wrap limit needs to be adjusted? 88 or 92 are commonly what I see in other languages.  Github's lines are I think 80 characters wide.

Reply:
The logic here doesnt make sense. opened  https://github.com/brown-ccv/VR-Volumeviewer/issues/49. I reformatted the file.